server/tests/security/conftest.py

```
@@ -0,0 +1,24 @@
+
+def sig_blocks(cmd: str) -> bool:
+    """Return True if the signature matcher catches *cmd*."""
+    return check_signature(cmd).matched
+
+
+def deny_blocks(cmd: str) -> bool:
+    """Return True if any universal deny rule matches *cmd*."""
+    for raw in _UNIVERSAL_DENY_RULES:
+        if re.search(raw["pattern"], cmd):
+            return True
+    return False
```

Ruff (0.15.12)

[warning] Use `return any(re.search(raw["pattern"], cmd) for raw in _UNIVERSAL_DENY_RULES)` instead of `for` loop

Replace with `return any(re.search(raw["pattern"], cmd) for raw in _UNIVERSAL_DENY_RULES)`

(SIM110)